css/style.css

```diff
@@ -1,7 +1,316 @@
+html {
+	box-sizing: border-box; /*padding и border включаются в общую ширину и высоту элемента*/
+ }
+
+*, *::before, *::after {
+	box-sizing: inherit;
+}
+
+html. body {
+	height: 100%;
+}
+
+body {
+		margin: 0 auto; 
+	color: #E8EEE4; 
+	font: normal 22px/26px MuseoSansCyrl;
+	overflow: scroll;
+}
+
+.wrap {
+	max-width: 1920px;
+	height: 100vh;
+	background: url(../img/iswanto-arif-448003-unsplash.png) no-repeat center; /*фоновая картинка*/
+	background-size: cover; /*картинка полностью покрывает блок при любых размерах*/
+	margin: 0 auto;
+}
+
+/*header styles start*/
+
+.header {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	padding: 5vh;
+	margin-bottom: 9vh;
+}
+
+.header__logo {
+	margin-left: 5vw;
+}
+
+.header__logo img {
+	max-width: 100%;
+	max-height: 100%;
+}
+
+.header__menu {
+	display: flex;
+	justify-content: flex-end;
+}
+
+.menu__link {
+	flex-wrap: nowrap;
+	color: #E8EEE4;
+	margin-right: 33.75px; /*отступ справа у пуктов меню*/
+	text-decoration: none; /*убрали нижнее подчеркивание*/
+}
+
+.menu__link:hover, .menu__link:focus, .menu__link:active { /*стили для наведения на ссылки меню*/
+	opacity: .50;
+}
+
+.header__burger { /*бургер-меню*/
+	display: none;
+	width: 25px;
+	height: 17px;
+	position: relative;
+}
+
+.burger__line {
+	display: block;
+	width: 100%;
+	height: 3px;
+	background: #FCF5ED;
+	position: absolute;
+	left: 0;
+}
+
+.burger__line_first {
+	top: 0;
+}
+
+.burger__line_second {
+	top: 50%;
+	transform: translateY(-50%);
+}
+
+.burger__line_third {
+	bottom: 0;
+}
+/*header styles end*/
+
+
+/*content styles start*/
+
+.content {
+	display: flex;
+	justify-content: space-between;
+}
+
+.info {
+	flex: 1; /*ширина флекс элемента*/
+	justify-content: center; /*выравнивание элементов по ширине - центр*/
+	display: flex;
+	flex-flow: column; /*flex-элементы в виде блоков друг над другом*/
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project-Bali/pullRequest?prid=5455736)